RevenueCatUI/Data/Variables.swift

```diff
@@ -85,6 +87,7 @@ private extension VariableDataProvider {
 
     func value(for variableName: String, locale: Locale) -> String {
         switch variableName {
+        case "app_name": return self.applicationName
```

Comment:
I made this a "secret" variable so the default template can display the app name as a title.
I don't think it makes sense to expose it in the docs / frontend, since users can just write whatever their app name is there, it doesn't have to be dynamic.

Thoughts @RevenueCat/cashnip?